src/svelte/src/components/Header/fieldsets/Settings.svelte

```diff
@@ -28,8 +28,22 @@ limitations under the License.
   } from '../../../stores'
   import FlexContainer from '../../layouts/FlexContainer.svelte'
   import { UIThemeCSSClass } from '../../../utilities/colorScheme'
+  import {MessageCommand} from "src/utilities/message";
 
   $: $bytesPerRow = $displayRadix === RADIX_OPTIONS.Binary ? 8 : 16
+
+  window.addEventListener('message', (msg) => {
+    switch (msg.data.command) {
+      case MessageCommand.fileInfo:
+      {
+        if ('byteOrderMark' in msg.data.data) {
+            const { byteOrderMark } = msg.data.data
+            if (byteOrderMark === 'UTF-8') $editorEncoding = 'utf-8'
```

Comment:
FE FF is for UTF-16BE, and EF BB BF is the UTF-8 BOM.   UTF-8 can typically be _assumed_, but it can also be _explicit_ with the UTF-8 BOM (though you're right that byte order isn't even a thing in UTF-8 as it is in UTF-16 and UTF-32).